tests/unit/ovirtlago/test_runtest.py

```diff
@@ -19,12 +19,15 @@ def dummy_test():
     return DummyTest(id=lambda: 123)
 
 
-@pytest.mark.parametrize('exc', [ExtractPathError, ExtractPathNoPathError])
+@pytest.mark.parametrize(
```

Comment:
I think that the commit message should explain why this change is needed, I guess it's because the interface of the exceptions was changed in lago.